src/workbench/SideBar.tsx

```diff
@@ -124,17 +124,26 @@ const SideBar = ({
     if (tabIndex !== -1) {
       onTabIndexChange(tabIndex);
       onSidebarExpand();
-      if (!api && !reference && !idea) {
+      if ((!api && !reference && !idea) || focusTab) {
```

Comment:
I don't remember why we need this -> `!api && !reference && !idea`? If we didn't do this check, we wouldn't need `focusTab`.

Reply:
My initial thought is that we were attempting to leave focus management to the code that handles the anchor for a specific item. But... that code doesn't actually focus. So I'm tentatively on board with removing the guard. I'll try it out.

Reply:
Removing these breaks focusing the panel when clicking a tab.

Reply:
I've kept the focus as part of the navigation state. I've tried to switch to using the identity of the params as an indication of when a navigation has occurred so we can know it's safe to focus if requested.